Majasdarbs/02-majasdarbs/Form/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Registration form</title>
+  </head>
+
+  <body>
+      <h2>Reģistrācijas forma</h2>
+        <fieldset>
+            <label for="name">Ievietot vārdu:</label>
+            <input type="text" name="name" id="name" required>
+            <label for="password">Ievietot paroli:</label> 
+            <input type="password" name="password" id="password" required>
+        </fieldset>
+     <br>
+        <fieldset>
+          <legend> <p style=background-color:yellow>Informācija par sevi</p> </legend>
+          <label for="dzimums">
+         <h4>Dzimums</h4>
+         </label> 
+
+         <input type="radio" name="Vīrietis" id="dzimums" required>
+         <label for="Vīrietis">Vīrietis</label>
+         <br>
+         <input type="radio" name="Sieviete" id="dzimums" required>
+         <label for="Sieviete">Sieviete</label>
```

Comment:
Need same name, because now you can choose both in this list. But need only one (vir vai siev)

Reply:
Please show example.